test/WebJobs.Script.Tests.Integration/TestFunctionHost.cs

```diff
@@ -269,7 +282,7 @@ public void Dispose()
             _testServer.Dispose();
         }
 
-        private async Task<bool> IsHostStarted(HttpClient client)
+        internal virtual async Task<bool> IsHostStarted()
```

Comment:
Made it virtual to temporarily override so that the test can allow Host to return even when it's `Initialized`